ocean_lib/common/ddo/service.py

```diff
@@ -8,6 +8,7 @@
 """
 import copy
 import logging
+from typing import Any, Dict, Optional, Tuple
 
 logger = logging.getLogger(__name__)
 
```

Comment:
Add `@enforce_types` to the `Service` class

Reply:
all functions except __init__ have `@enforce_types` due to the issue with Service and ServiceAgreement inheritance. It spews out the error about arguments (surely it is somewhere in the resolved issues). For __init__ I can not add it, so I can not add it on class-level, only on individual functions.